app/assets/javascripts/taxonomy-select.js

```diff
@@ -3,68 +3,70 @@
+    this.$el = options.$el
+    this.options = this.instantiateOptions()
   }
 
-  TaxonomySelect.prototype.update = function updateTaxonomyFacet(){
-    this.disableSubTaxonFacet();
-    this.resetSubTaxonValue();
-    this.showRelevantSubTaxons();
-  };
+  TaxonomySelect.prototype.update = function updateTaxonomyFacet () {
+    this.disableSubTaxonFacet()
+    this.resetSubTaxonValue()
+    this.showRelevantSubTaxons()
+  }
+
+  TaxonomySelect.prototype.$topLevelTaxon = function $topLevelTaxon () {
+    return this.$el.find('#level_one_taxon')
+  }
 
-  TaxonomySelect.prototype.$topLevelTaxon = function $topLevelTaxon() {
-    return this.$el.find('#level_one_taxon');
-  };
+  TaxonomySelect.prototype.$subTaxon = function $subTaxon () {
+    return this.$el.find('#level_two_taxon')
+  }
 
-  TaxonomySelect.prototype.$subTaxon = function $subTaxon() {
-    return this.$el.find('#level_two_taxon');
-  };
+  TaxonomySelect.prototype.disableSubTaxonFacet = function disableSubTaxonFacet () {
+    var topLevelTaxonSelected = !!this.$topLevelTaxon().val()
+    this.$subTaxon().attr('disabled', !topLevelTaxonSelected)
+  }
 
-  TaxonomySelect.prototype.disableSubTaxonFacet = function disableSubTaxonFacet() {
-    var topLevelTaxonSelected = !!this.$topLevelTaxon().val();
-    this.$subTaxon().attr('disabled', !topLevelTaxonSelected);
-  };
+  TaxonomySelect.prototype.showRelevantSubTaxons = function showRelevantSubTaxons () {
+    var taxons = this.options[this.$topLevelTaxon().val()]
 
-  TaxonomySelect.prototype.showRelevantSubTaxons = function showRelevantSubTaxons() {
-    var taxons = this.options[this.$topLevelTaxon().val()],
-        subtaxon = this.$subTaxon();
+    var subtaxon = this.$subTaxon()
 
-    subtaxon.find('option').each(function(){
-      if ($(this).val()) { $(this).remove(); }
+    subtaxon.find('option').each(function () {
+      if ($(this).val()) { $(this).remove() }
     })
 
-    subtaxon.append(taxons);
-  };
+    subtaxon.append(taxons)
+  }
 
-  TaxonomySelect.prototype.instantiateOptions = function() {
-    var options = {};
+  TaxonomySelect.prototype.instantiateOptions = function () {
+    var options = {}
 
-    this.$subTaxon().find('option').each(function(){
-      var parent = $(this).attr('data-topic-parent');
+    this.$subTaxon().find('option').each(function () {
+      var parent = $(this).attr('data-topic-parent')
 
       options[parent] = options[parent] || []
       options[parent].push(this)
-    });
+    })
 
-    return options;
+    return options
   }
 
-  TaxonomySelect.prototype.resetSubTaxonValue = function resetSubTaxonValue() {
-    var selected = this.$subTaxon().find(':selected'),
-        parentTaxon = this.$topLevelTaxon().val(),
-        isOrphanedSubTaxon = selected && selected.attr('data-topic-parent') !== parentTaxon;
+  TaxonomySelect.prototype.resetSubTaxonValue = function resetSubTaxonValue () {
+    var selected = this.$subTaxon().find(':selected')
+
```

Comment:
Nitpick: we probably don't need a newline between each variable declaration.

Reply:
See https://github.com/alphagov/finder-frontend/pull/1086#discussion_r280335766